### DISCUSSION TITLE
Unable to build insiders image for arm (Apple M2)

### DISCUSSION THREAD

**Discussion by tomas-intestudios** [squidfunk/mkdocs-material#5549] [General]
Hello, 

I followed insiders guid to be able to build insiders docker image. I was able to build & pull linux (amd64) docker image, but I cannot get arm image.

```
$ docker pull ghcr.io/${username}/mkdocs-material-insiders:9.1.10-insiders-4.33.0
9.1.10-insiders-4.33.0: Pulling from ${username}/mkdocs-material-insiders
no matching manifest for linux/arm64/v8 in the manifest list entries
```

I found out, that arm architecture is commented out in build gh workflow.

I also tried older tags (9.0.x), it looks like arm image was built, but workflow failed about some unauthorized issue, when workflow was attempting to push image into container repository.

**Comment by squidfunk** [comment]
> I found out, that arm architecture is commented out in build gh workflow.

Jup, that's correct. You need to comment it in. The reason why it's commented out is that build times are much higher when building ARM images and we don't want to impose long build times onto all of our users, as those also incur higher cost when running on private GitHub Actions. Thus, it's opt-in. Just comment it in and you should be good to go!
